frame/executive/src/lib.rs

```diff
@@ -256,19 +250,10 @@ where
 	/// This should only be used for testing.
 	#[cfg(feature = "try-runtime")]
 	pub fn try_runtime_upgrade() -> Result<frame_support::weights::Weight, &'static str> {
-		<
-			(frame_system::Pallet::<System>, COnRuntimeUpgrade, AllPallets)
-			as
-			OnRuntimeUpgrade
-		>::pre_upgrade().unwrap();
-
+		<(COnRuntimeUpgrade, AllPallets) as OnRuntimeUpgrade>::pre_upgrade().unwrap();
```

Comment:
I think this is also a noteworthy change. Here system pallet upgrades after the runtime upgrades.

Dunno actually if this can cause issues?

Reply:
I think for pre_upgrade/post_upgrade the order wasn't so important as it is just checks.
But the order of on_runtime_upgrade is important. That said the order was: custom, then system then pallets.
I will precise this in the note

Reply:
`try_on_runtime_upgrade` should always imitate what the real on_runtime_upgrade would do, which is respected here, so no issues.